README.md

```diff
@@ -3,33 +3,130 @@
+and Windows, and download `configlet` to your local drive. Find more information 
+about [configlet][git-configlet] [here][git-configlet].
 
-    make test
-    
-Note: `make test` is recommended BEFORE submitting a PR.  It will test your submission, and help guard against unintentional, unrelated changes.
-#### Test Specific Assignment
-Pass the exercise name to make script to run the tests for a specific exercise:
+### Scripts
 
-    make test-assignment ASSIGNMENT=hello-world
-    
-## Contributing Guide
-For an in-depth discussion of how exercism language tracks and exercises work, please see [CONTRIBUTING.md](https://github.com/exercism/javascript/blob/master/CONTRIBUTING.md)
+We have various `scripts` for you in order to aid with maintaining and 
+contributing to this repository.
+
+#### `lint`
+
+```js
+/*
+ * Run this script (from root directory): npx @babel/node scripts/lint
+ *
+ * This runs `eslint` on all sample solutions (and test) files
+ */
+```
+
+If the `ASSIGNMENT` environment variable is set, only _that_ exercise is tested. For
+example, if you only want to lint `two-fer`, you may, depending on your environment
+use:
+
+```shell
+ASSIGNMENT=two-fer npx @babel/node scripts/lint
+```
+
+#### `test`
+
+```js
+/**
+ * Run this script (from root directory): npx @babel/node scripts/test
+ *
+ * This runs `jest` tests for all sample solutions
+ */
+```
+
+If the `ASSIGNMENT` environment variable is set, only _that_ exercise is tested. For
+example, if you only want to test the `example.js` for `two-fer`, you may, depending 
+on your environment, use:
+
+```shell
+ASSIGNMENT=two-fer npx @babel/node scripts/test
+```
+
+#### `sync`
+
+```js
+/**
+ * Run this script (from root directory): npx @babel/node scripts/sync
+ *
+ * This script is used to propagate any change to root package.json to
+ * all exercises and keep them in sync.
+ * There is a CI step which checks that package.json in root & exercises match
+ * (see checksum script for more info).
+ */
+```
+
+If the `ASSIGNMENT` environment variable is set, only _that_ exercise is tested. For
+example, if you only want to sync the files for `two-fer`, you may, depending on 
+your environment, use:
+
+```shell
+ASSIGNMENT=two-fer npx @babel/node scripts/sync
+```
+
+#### `checksum`
+
+```js
+/*
+ * Run this script (from root directory): npx @babel/node scripts/checksum
+ *
+ * This will check root `package.json` matches each exercise's `package.json`.
+ * But the catch is there are some dependencies used for build but not served to end users
+ * We skip those dependencies while performing checksum.
+ * See `SKIP_PACKAGES_FOR_CHECKSUM` in helpers.js for list of skipped packages.
+ */
+```
+
+#### `ci`
+
+```js
+/**
+ * Run this script (from root directory): npx @babel/node scripts/ci
+ *
+ * This will run following checks:
+ * 1. Check config in all exercises matches
+ * 2. Run eslint to check code-style
+ * 3. Run tests against sample solutions
+ */
+```
```

Comment:
These scripts are described in `CONTRIBUTING.md` (which is linked below) - I feel we don't need this elaborate description here, maybe have a shorter version to run `ci` command here and more details in contributing doc?

Reply:
Yeah I like.

Reply:
See push!